internal/pkg/store/store.go

```diff
@@ -20,37 +20,37 @@ import (
 	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
 )
 
-// Parameter name formats for resources in a project. Projects are laid out in SSM
+// Parameter name formats for resources in an application. Applications are laid out in SSM
 // based on path - each parameter's key has a certain format, and you can have
-// hierarchies based on that format. Projects are at the root of the hierarchy.
-// Searching SSM for all parameters with the `rootProjectPath` key will give you
-// all the project keys, for example.
+// hierarchies based on that format. Applications are at the root of the hierarchy.
+// Searching SSM for all parameters with the `rootApplicationPath` key will give you
+// all the application keys, for example.
 
 // current schema Version for Projects
 const schemaVersion = "1.0"
 
 // schema formats supported in current schemaVersion. NOTE: May change to map in the future.
 const (
-	rootProjectPath  = "/ecs-cli-v2/"
-	fmtProjectPath   = "/ecs-cli-v2/%s"
-	rootEnvParamPath = "/ecs-cli-v2/%s/environments/"
-	fmtEnvParamPath  = "/ecs-cli-v2/%s/environments/%s" // path for an environment in a project
-	rootAppParamPath = "/ecs-cli-v2/%s/applications/"
-	fmtAppParamPath  = "/ecs-cli-v2/%s/applications/%s" // path for an application in a project
+	rootApplicationPath = "/copilot/applications/"
+	fmtApplicationPath  = "/copilot/applications/%s"
+	rootEnvParamPath    = "/copilot/applications/%s/environments/"
+	fmtEnvParamPath     = "/copilot/applications/%s/environments/%s" // path for an environment in an application
+	rootSvcParamPath    = "/copilot/applications/%s/services/"
+	fmtSvcParamPath     = "/copilot/applications/%s/services/%s" // path for a service in an application
```

Comment:
```suggestion
	rootSvcParamPath    = "/copilot/applications/%s/components/"
	fmtSvcParamPath     = "/copilot/applications/%s/components/%s" // path for a service in an application
```

Eventually, if we ever expand to other types like "workflows", "jobs" we want to have an agnostic way of looking those application components up.